src/components/cards/cardSummary/cardSummary.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import { AvatarIcon } from '../../avatars';
+import { Button } from '../../button';
+import { IconType } from '../../icon';
+import { Card } from '../card';
+import type { ICardSummaryProps } from './cardSummary.api';
+
+export const CardSummary: React.FC<ICardSummaryProps> = (props) => {
+    const { icon, value, description, action, isStacked = true, className, ...otherProps } = props;
+
+    const containerClassNames = classNames(
+        'grid grid-cols-[auto_max-content] items-center gap-4 p-4 md:px-6 md:py-5',
+        { 'grid-cols-[auto_max-content] md:gap-5': isStacked },
+        {
+            'grid-cols-[auto_max-content] md:grid-flow-col md:grid-cols-[auto_1fr_1fr_max-content] md:gap-6':
+                !isStacked,
+        },
+    );
+
+    return (
+        <Card
+            className={classNames(
+                { 'w-[320px] md:w-[480px]': isStacked },
+                { 'w-[320px] md:w-[640px]': !isStacked },
```

Comment:
Should the width be fixed or does it take the size of the parent container?

Reply:
All these containers have a fixed size set on Figma

Reply:
Yeah, I'm asking myself if that's the actual card size or if this is just Selim needing to limit it to a size on Figma 🤔